plugins/mcreator-localization/help/default/command/when_executed.md

```diff
@@ -1,3 +1,5 @@
 When the command is executed, the selected procedure here will be executed.
 
-Commands usually run on the server-side only.
\ No newline at end of file
+Commands usually run on the server-side only.
+
+Note: You should only use this procedure selector if you do not need parameters.
```

Comment:
Consider removing procedure selector altogether in favor of new builder. One can use call procedure procedure block to call procedure from there. Also prepare migration path for existing workspaces for this.